doc/reference/network_bridge.md

```diff
@@ -6,26 +6,25 @@ discourse: 7322
 # Bridge network
 
 As one of the possible network configuration types under LXD, LXD supports creating and managing network bridges.
-L<!-- Include start bridge intro -->
```

Comment:
When was that typo introduced? If it was an earlier commit in this branch, it'd be better to fix that earlier commit to make this one's diff less confusing (and less likely to conflict on cherry-picks).